### ISSUE TITLE
[question] Unable to do hot reload, missing node script?

### ISSUE THREAD

**Issue by francescotaioli** [cezerin/cezerin#560]
hi, i want to modify the teme such as the cartItem Icon and add some features. I want to to the hot reload, so i'm looking for the  theme:build:dev script ( show in [docs](https://github.com/cezerin/cezerin/tree/master/docs)

theme:build:dev | Refresh theme after modification and watch.

After launch node start and this script, in teory, i'm able to do the hot reload. the problem is that this script doesn't exist in package json. Any idea and any help?

**What script i have to launch and in what order?**

thanks to all and thanks for cezerin @sonufrienko

**Comment by francesco-taioli** [comment]
i find this question, remains like this? https://github.com/cezerin/cezerin/issues/385

**Comment by selmi-karim** [comment]
same issue

**Comment by shsunmoonlee** [comment]
I'm trying to change the architecture on this one, and add webpack hot module replacment on react part. Currently, react code is converted to html and rendered from the node.js server. I think it's better to use react-boilerplate architecture with separate nodejs server just for api, or next.js can be an option. If anyone wants to collaborate on re-do the architecture, message me.

**Comment by neunygph** [comment]
I agree, the server api, even the admin site should be on separated projects. They can be on a single project for example project but it's not an ideal solution if the business grow bigger. For instance, if we have too many request on the store site, it does not make sense if the admin and api server got hit too But this is just a theory, it would be a big change to the architecture.

**Comment by shsunmoonlee** [comment]
So I re wrote the script and still have to figure out how to see changes in local npm module 'theme' show up without rebuilding, re-babel-ling. Currently theme has to be babel-led into dist/index.js before it gets served as local npm package.
